src/Orc.Theming/Services/Interfaces/IBaseColorSchemeService.cs

```diff
@@ -13,6 +13,12 @@ public interface IBaseColorSchemeService
         /// <returns>at least one base color, otherwise GetBaseColor will throw an exception</returns>
         IReadOnlyList<string> GetAvailableBaseColorSchemes();
 
+        /// <summary>
+        /// returns the available base colors image uris (at least one)
+        /// </summary>
+        /// <returns>at least one base color image uris, otherwise GetBaseColor will throw an exception</returns>
+        IReadOnlyList<KeyValuePair<string, string>> GetAvailableBaseColorSchemeUris();
```

Comment:
Remove, this is purely for representation purposes only and should not be part of the scheme service. In this case only the view / view model need to be aware that the state is being represented as images and thus only need to be available there.